ingestion/src/metadata/ingestion/source/metabase.py

```diff
@@ -57,7 +57,7 @@ class MetabaseSourceConfig(ConfigModel):
     chart_pattern: IncludeFilterPattern = IncludeFilterPattern.allow_all()
     service_name: str
     service_type: str = DashboardServiceType.Metabase.value
-    database_service_name: str = None
+    db_service_name: str = None
```

Comment:
Let's wrap this as `Optional[str]` to get rid of the smell

Reply:
@ayush-shah can you take care of this.

Reply:
yes, on it.